softlayer/resource_softlayer_virtual_guest.go

```diff
@@ -719,6 +720,7 @@ func resourceSoftLayerVirtualGuestRead(d *schema.ResourceData, meta interface{})
 	d.Set("private_network_only", *result.PrivateNetworkOnlyFlag)
 	d.Set("hourly_billing", *result.HourlyBillingFlag)
 	d.Set("local_disk", *result.LocalDiskFlag)
+	d.Set("post_install_script_uri", *result.PostInstallScriptUri)
```

Comment:
When resourceSoftLayerVirtualGuestRead function is invoked, `result.PostInstallScriptUri` returns `nil` and the terraform generates panic status. Please check if SL API returns `PostInstallScriptUri` properly. And post_install_script_url is an optional attribute. It is necessary to check if `result.PostInstallScriptUri` is `nil` or not in advance.